app/Scripts/BoidAbsorptionComponent.gd

```diff
@@ -0,0 +1,23 @@
+extends Node
+
+onready var _enemy_manager = get_node("/root/Main/EnemyManager")
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+var _player: Node 
```

Comment:
created this piece of logic as a 'component' Node under the Player node. This is a pattern we can follow to organize logic into separate scripts